Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/RayTracing/RayTracingSceneSrg.azsli

```diff
@@ -163,4 +161,4 @@ ShaderResourceGroup RayTracingSceneSrg : SRG_RayTracingScene
     // - Optional stream buffers such as Tangent, Bitangent, and UV are indicated in the MeshInfo.m_bufferFlags field
     // - Buffers for a particular mesh start at MeshInfo.m_bufferStartIndex
     ByteAddressBuffer m_meshBuffers[];
-}
\ No newline at end of file
+}
```

Comment:
I saw no newline warning in RayTracingMaterialSrg.azsli as well. We can fix them altogether.